tests/manual-test-cases/Group18-VIC-UI/setup-testbed.robot

```diff
@@ -236,7 +236,7 @@ Deploy Nimbus ESXi Server For NGC Testing
     [Return]  ${user}-${name}  ${ip}
 
 Deploy Nimbus vCenter Server For NGC Testing
```

Comment:
sorry should have been more clear, we are deprecating the standard ESX and VC deploys in favor of async deploys, so overloading the name field for VC deploy would be:

Deploy Nimbus vCenter Server Async  UITEST-VC%{TEST_VSPHERE_VER}-rand --useQANgc

(note single space between name and useQANgc)